src/main/resources/alfresco/transformers/0140-misc.json

```diff
@@ -0,0 +1,65 @@
+{
```

Comment:
This file is no longer needed if the new misc T-Engine replies with the same value from .../transform/config, so it should be removed.